mindsdb/libs/phases/data_splitter/data_splitter.py

```diff
@@ -1,20 +1,31 @@
-from mindsdb.config import CONFIG
-from mindsdb.libs.constants.mindsdb import *
-from mindsdb.libs.phases.base_module import BaseModule
-from mindsdb.libs.data_types.mindsdb_logger import log
+from mindsdb.mindsdb.config import CONFIG
+from mindsdb.mindsdb.libs.constants.mindsdb import TRANSACTION_LEARN
```

Comment:
I needed to prepend every import statement with `mindsdb` to overcome an import error -- can be removed if needed.

Reply:
This seems to be breaking travis, fixing